modules/gapi/include/opencv2/gapi/streaming/onevpl/cfg_params.hpp

```diff
@@ -110,6 +110,62 @@ struct GAPI_EXPORTS CfgParam {
     static CfgParam create_implementation(uint32_t value);
     static CfgParam create_implementation(const char* value);
 
+
+    static constexpr const char *vpp_frames_pool_size_name() { return "vpp_frames_pool_size"; }
```

Comment:
it's ascending to Source itself but not for preprocessing settings: It's parameter similar with `decode_frames_pool_size` old one - and  according to thin optimization  settings

Reply:
So, it has to be this?  
I thought it is just a name which you use across the code, so you can practically choose anything as a name. I just suggested to keep it similar, is it possible?

Reply:
i'm sorry, maybe i was confusing:

> "vpp.FramesPoolSize"

you suggest to name is as "vpp.In.Width", right?
It should not, because params  have different  meaning: "vpp.In.Width" involved in preprocessing, it's configuration param and it used to fill `mfxVideoParam` structure for VPL api; but `vpp_frames_pool_size` determines how much surfaces we  should allocate to make preprocessing fast or minimal memory consumption - it's some kind of optimization params. 
Maybe we should implement different param classes at least: config & optimization. But  before do this, I wish I have get user experience feedback :)

Reply:
OK!